### DISCUSSION TITLE
A new repoitory for Hacktoberfest 2023

### DISCUSSION THREAD

**Discussion by ossamamehmood** [ossamamehmood/Hacktoberfest#6494] [Announcements]
We have created a new repository : 
https://github.com/ossamamehmood/Hacktoberfest2023

Note: This current repo was excluded due to high volume spam and low-quality contribution. So, we set up a new repo that follows #hacktoberfest guidelines.

Additionally, we acknowledge fair and quality contributions. You must follow Hacktooberfest contribution guidelines to approve your Pull Request!

Special thanks to our maintainer's team who are always trying to improve the quality of Hacktoberfest events from now onwards.

**Comment by Mrk-Mzj** [comment]
The new repository is excluded as well, I'm afraid.
